src/busco/script.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+## VIASH START
+## VIASH END
+
+
+[[ "$par_tar" == "false" ]] && unset par_tar
+[[ "$par_force" == "false" ]] && unset par_force
+[[ "$par_offline" == "false" ]] && unset par_offline
+[[ "$par_opt_out_run_stats" == "false" ]] && unset par_opt_out_run_stats
+[[ "$par_quiet" == "false" ]] && unset par_quiet
+[[ "$par_restart" == "false" ]] && unset par_restart
+[[ "$par_auto_lineage" == "false" ]] && unset par_auto_lineage
+[[ "$par_auto_lineage_euk" == "false" ]] && unset par_auto_lineage_euk
+[[ "$par_auto_lineage_prok" == "false" ]] && unset par_auto_lineage_prok
+[[ "$par_augustus" == "false" ]] && unset par_augustus
+[[ "$par_long" == "false" ]] && unset par_long
+[[ "$par_scaffold_composition" == "false" ]] && unset par_scaffold_composition
+[[ "$par_miniprot" == "false" ]] && unset par_miniprot
+
+if [[ -n "$par_output_prefix" ]]; then 
+    prefix="$par_output_prefix"
+else 
+    prefix="$(basename -- $par_input)"
+fi
+
+busco \
+    --in "$par_input" \
+    --mode "$par_mode" \
+    --out "$prefix" \
```

Comment:
```suggestion
tmp_dir=$(mktemp -d -p "$meta_temp_dir" busco_XXXXXXXXX)

busco \
    --in "$par_input" \
    --mode "$par_mode" \
    --out "" \
    --out_dir "$tmp_dir" \
```

Reply:
with slight modifications - having --out as an empty string gave trouble (because it also tries to create a folder) - modified it to be a random hex